openassessment/xblock/resolve_dates.py

```diff
@@ -46,7 +58,11 @@ def _parse_date(value, _):
         try:
             return parse_date(value).replace(tzinfo=pytz.utc)
```

Comment:
Cool. I figured it was alright, just wanted to make sure nothing slipped through the cracks.